src/asserts/big-number-equal-to-assert.js

```diff
@@ -3,19 +3,22 @@
  * Module dependencies.
  */
 
-import { Violation } from 'validator.js';
+import BigNumberAssert from './big-number-assert';
+import { Assert as BaseAssert, Violation } from 'validator.js';
```

Comment:
We shouldn't need to rename `Assert`, as `Assert.extend()` doesn't mutate it. Or was there another reason?

Reply:
Just because every other file that imports `Assert` renames it to `BaseAssert`.

Reply:
That seems to be because they later do:

```js
const Assert = BaseAssert.extend({ ... });

new Assert().Foobar().validate();
```

While here we're doing:

```js
const is = BaseAssert.extend({ ... });

is.foobar().validate();
```

So, in hindsight, to keep consistent, we should probably adapt these changes to be more like those other usages.

Reply:
You're right, I've update the PR to use the codebase convention.